framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -289,20 +290,23 @@ public int runTests() throws Exception {
       if (cmdParam.trackMemory) {
     	  queryMemoryUsage();
       }
-
+      if(tempTobeRunFailures.size()>0){
+        tempTobeRunFailures.clear();
```

Comment:
might not be necessary. Please check.

Reply:
Checked abhishek ,I think this is necessary and in fact needs to be added to the other data structure as well (passing tests,execution failures etc),when running for 2 iterations they pile on the existing data structure and produce duplicates :- 

Plan Verification Failures:

/root/drillAutomation/drill-test-framework/framework/resources/Functional/filter/pushdown/plan/q4.sql
/root/drillAutomation/drill-test-framework/framework/resources/Functional/filter/pushdown/plan/q4.sql

Total Tests                  : 8397
Passing Tests                : 16756 --(the number is more than the total tests)
Failing Tests                : 38

> Execution Failures         : 36 --(this just doubled up)
> Data Verification Failures : 0
> Plan Verification Failures : 2 --(this just doubled up)
> Timeout Failures           : 0

Reply:
should I make the changes for them as well ?